api/overdrive.py

```diff
@@ -991,8 +991,10 @@ def circulation_lookup(self, book):
             )
             book = dict(id=book_id)
         else:
-            book_id = book['id']
             circulation_link = book['availability_link']
+            # Make sure we use v2 of the availability API,
+            # even if Overdrive gave us a link to v1.
+            circulation_link = self.make_link_safe(circulation_link)
```

Comment:
There may be other places in overdrive.py where we need to make this change, but this is the only place where we reference 'availability_link'.

This bit of the API client is actually the first code I wrote for this project, and it's a mess that needs refactoring... but not today.